### PR TITLE
Porting ExampleGallery and SimpleSample from UAP to WAS

### PR DESCRIPTION
**Pull request by daamenik**
This PR involves updating the old UAP ExampleGallery and SimpleSample to WindowsAppSDK/Reunion version 1.0.0.27-preview. 

In the ExampleGallery project, just over a third of the samples contain Win2D elements like CanvasAnimatedControl and InkCanvas, which are not yet supported for WinUI3. Thus, those samples are contained within the ExampleGallery folder, but excluded from the project.

Much of the actual changes made in this repo are Visual Studio configurations and small changes from Windows.UI.* to Microsoft.UI.*, but some examples used APIs that have been phased out in WAS, so changes had to be made there as well. The comparison should be fairly easy to navigate due to the relatively small size of these changes. 

Note: there is a known bug in the ExampleGallery that has been registered internally. The VisualState element (I think) in XAML is causing the app to crash when changing desktop displays.

### CODE REVIEW

**Review by iablaauw-MS — COMMENTED** [review]
For `ExampleGallery (Package)/ExampleGallery (Package).assets.cache` it looks like this should probably be part of .gitignore?

**Review by getrou — COMMENTED** [review]
Is it possible to leave the AnimatedControlExample.xaml and AnimatedControlExample.xaml.cs files where they are, but simply exclude them from the VS project?  Deleting them also deletes their history.

**Comment by daamenik** [comment]
> Is it possible to leave the AnimatedControlExample.xaml and AnimatedControlExample.xaml.cs files where they are, but simply exclude them from the VS project? Deleting them also deletes their history.

Yeah I can totally put them back. Forgot that I deleted them; must have been a long time ago

**Comment by daamenik** [comment]
> For `ExampleGallery (Package)/ExampleGallery (Package).assets.cache` it looks like this should probably be part of .gitignore?

Got this deleted and put in the gitignore

File: ExampleGallery/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (iablaauw-MS) [inline ExampleGallery/Properties/launchSettings.json:1]: This file should also probably go under gitignore, but I'm less sure. VS seems to create it for me when I change some project settings, but maybe we want to persist that.

File: README.md [hunk 2]
```diff
@@ -2,40 +2,38 @@
 
 These sample projects show how to use the Win2D graphics API.
 
-- **Example Gallery** is the main Win2D demo and sample application. This is a C# XAML 
+- This gallery is currently being updated to use **ProjectReunion**.  UAP is in preview in ProjectReunion.0.5 
+  Because these samples are UAP, they are using **ProjectReunion.0.5.0-prerelease**
```
Comment (iablaauw-MS) [inline README.md]: Is this still on Reunion 0.5 or did it get updated to 0.8 at least?

File: README.md [hunk 3]
```diff
@@ -2,40 +2,38 @@
...
 
-- **CoreWindow Example** is a C# application that draws directly to a CoreWindow, 
+- [Not yet updated for ProjectReunion] **CoreWindow Example** is a C# application that draws directly to a CoreWindow, 
   without using XAML.
 
 - **Simple Sample** is, yup, you guessed it! Pretty much the simplest possible app that 
   uses Win2D - it's our version of the venerable "hello world".
 
-- **[Win2D-MazeGame](https://github.com/microsoft/Win2DMazeGame)** (in a separate repository)
+- **Simple Smaple Desktop** is the same version of SimpleSample, but updated to be a Reunion Desktop instead of a UAP.
```
Comment (iablaauw-MS) [inline README.md]: Nit: spelling

File: SimpleSample (Package)/Package.appxmanifest [hunk 4]
```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (iablaauw-MS) [inline SimpleSample (Package)/Package.appxmanifest]: Looks like the *.assets.cache for this project snuck in as well.

**Review by iablaauw-MS — CHANGES_REQUESTED** [review]

File: ExampleGallery/Infrastructure/CustomThumbnails.cs [hunk 5]
```diff
@@ -59,26 +60,26 @@ IRandomAccessStream ICustomThumbnailSource.Thumbnail
         }
     }
 
-
+    // vvvvvvvv the below method is commented out because CustomControlExample is not supported in WinUI3 yet.
```
Comment (getrou) [inline ExampleGallery/Infrastructure/CustomThumbnails.cs]: vvvvvvv

Comment (daamenik) [reply ExampleGallery/Infrastructure/CustomThumbnails.cs]: those are supposed to be down arrows but that was probably a misstep

**Review by iablaauw-MS — APPROVED** [review]

**Review by iablaauw-MS — APPROVED** [review]

**Review by getrou — APPROVED** [review]

**Pull request merged**